README.md

```diff
@@ -418,8 +418,16 @@ TestName | How to enable | Example
 
 *Performance tests use Gatling. You can find more information about the tool on their website https://gatling.io/.
 
+You can passthrough options to the frontend security scan script by using the ZAP_URL_EXCLUSIONS variable in your Jenkinsfile. This allows you to customise zap proxy scans of your application.
```

Comment:
Docs need updating with new approach ?

Reply:
On hmcts.github.io you mean?

Reply:
I just meant the README. It says "You can passthrough options to the frontend security scan script by using the ZAP_URL_EXCLUSIONS variable in your Jenkinsfile" . shouldn't be an optional parameter to `enableSecurityScan` instead?

Reply:
Updated